components/automate-ui/src/app/pages/infra-proxy/chef-servers-list/chef-servers-list.component.html

```diff
@@ -0,0 +1,51 @@
+<app-client-runs-sidebar></app-client-runs-sidebar>
+
+<div class="container">
+    <main>
+      <chef-page-header>
+        <chef-heading>Chef Servers</chef-heading>
+        <chef-subheading>Manage Chef Servers with Chef Automate</chef-subheading>
+      </chef-page-header>
+
+      <app-create-chef-server-modal
+        [visible]="createModalVisible"
+        (close)="closeCreateModal()"
+        [conflictErrorEvent]="conflictErrorEvent"
+        (createClicked)="createChefServer()">
+      </app-create-chef-server-modal>
+
+      <div class="page-body">
+        <chef-toolbar>
+          <chef-button primary (click)="openCreateModal()">Add Chef Server</chef-button>
+        </chef-toolbar>
+        <chef-table>
```

Comment:
same as above - we now have `chef-table-new` and its corresponding table elements in non-stencil form